selectors-4/Overview.bs

```diff
@@ -36,6 +36,14 @@ spec:css-pseudo-4; type:selector;
 	text: ::first-line
 	text: ::first-letter
 </pre>
+<pre class="anchors">
+urlPrefix: https://html.spec.whatwg.org/multipage/; spec: HTML
```

Comment:
All of these definitions already exist in HTML and are reasonably marked up, they're just not exported.

Please swap to specifying these in link-defaults, and then probably ask HTML to export them.

Reply:
Fixed in 59f2bc1. Export of those terms requested in whatwg/html#6603.